tests/queries/0_stateless/03369_system_reload_config_exception.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+echo "Test 1: Check exception in local mode"
+# Run query in clickhouse-local, should fail with UNSUPPORTED_METHOD
+${CLICKHOUSE_LOCAL} --query "SYSTEM RELOAD CONFIG" 2>&1 | grep -F "SYSTEM RELOAD CONFIG query is not supported in clickhouse-local" && echo "OK" || echo "FAIL"
+
+echo "Test 2: Check STOP LISTEN HTTP exception in local mode"
+${CLICKHOUSE_LOCAL} --query "SYSTEM STOP LISTEN HTTP" 2>&1 | grep -F "SYSTEM STOP LISTEN HTTP query is not supported in clickhouse-local" && echo "OK" || echo "FAIL"
+
+echo "Test 3: Check START LISTEN HTTP exception in local mode"
+${CLICKHOUSE_LOCAL} --query "SYSTEM START LISTEN HTTP" 2>&1 | grep -F "SYSTEM START LISTEN HTTP query is not supported in clickhouse-local" && echo "OK" || echo "FAIL" 
```

Comment:
I think you have to handle thesse too. Could you please make sure this tests succeeds? Also, you can convert this into an SQL script:

```
SYSTEM RELOAD CONFIG -- { serverError UNSUPPORTED_METHOD };
SYSTEM STOP LISTEN HTTP -- { serverError UNSUPPORTED_METHOD };
SYSTEM START LISTEN HTTP -- { serverError UNSUPPORTED_METHOD };
```

This should work (after you fix `{STOP,START} LISTEN HTTP`